cypress/support/fragments/inventory/inventoryActions.js

```diff
@@ -43,20 +49,37 @@ export default {
     NewInventoryInstance.checkExpectedOCLCPresence(specialOCLCWorldCatidentifier);
   },
   verifySaveUUIDsFileName(actualName) {
-    // Check naming mask
     const expectedFileNameMask = /SearchInstanceUUIDs\d{4}-\d{2}-\d{2}T\d{2}_\d{2}_\d{2}\+\d{2}_\d{2}\.csv/gm;
     expect(actualName).to.match(expectedFileNameMask);
 
-    // Check date value
     const dateString = actualName.split('/');
     const actualDate = Date.parse(dateString[dateString.length - 1].slice(19, 38).replaceAll('_', ':'));
-    expect(actualDate).to.be.greaterThan(Date.now() - 100000);
-    expect(actualDate).to.be.lessThan(Date.now() + 100000);
+    verifyFileNameDate(actualDate);
   },
+
   verifySavedUUIDs(actualUUIDs, expectedUUIDs) {
     const formattedActualUUIDs = actualUUIDs.replaceAll('"', '').split('\n');
     for (let i = 0; i < expectedUUIDs.length; i++) {
       expect(expectedUUIDs[i]).to.eq(formattedActualUUIDs[i]);
     }
-  }
+  },
+
+  verifySaveSQLQueryFileName(actualName) {
+    const expectedFileNameMask = /SearchInstanceCQLQuery\d{4}-\d{2}-\d{2}T\d{2}_\d{2}_\d{2}\+\d{2}_\d{2}\.cql/gm;
+    expect(actualName).to.match(expectedFileNameMask);
+
+    const dateString = actualName.split('/');
+    const actualDate = Date.parse(dateString[dateString.length - 1].slice(22, 41).replaceAll('_', ':'));
```

Comment:
you can move date parser into DateUtils in appropriate method

Reply:
I thought about it, but I can't. I need a lot of custom values in slice and replace for parsing depends of file name. It can not be universal.

Reply:
parse(dateString[dateString.length - 1] - > parse(dateString[-1]

Reply:
you can name it as SomeSpecialParsingInSomeSpecialPlace()